lib/views/course_view/components/course_section.dart

```diff
@@ -43,8 +48,10 @@ class CourseSection extends StatelessWidget {
           _buildCourseSectionOrMessage(
             context: context,
             title: sectionTitle,
+            isCourseSection: isCourseSection,
             onViewAll: onViewAll,
             courses: courses ?? MockData.mockCourses,
+            streams: streams ?? [], //TODO add mock streams
```

Comment:
You can add it in the Mock file. easy to do :)

Reply:
Actually why do we even need the mock data at this point? 
Since we have `streams: streams ?? [], ` and then we check if `streams.isEmpty` we display a message that there are no livestreams - where does the mock data come in?

Reply:
yeah i thought you wrote the TODO. In this case remove the comment true